### ISSUE TITLE
Midpoint Review - Accessibility Feedback - BTSSS (Travel Pay) - MVP

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#80191]
## Next Steps for the VFS team

- [x] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Really love this prototype. Thank you!
- Very happy to hear you're already testing with a screen reader!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Should:** The "Showing X - Y of Z events" text should appear before the sort dropdown.
- [ ] **Should:** In similar experiences on VA.gov, the "Showing X - Y of Z" text is coded as an H2 (styled as plain text). That makes it a little easier for screen reader users to navigate quickly to the list of events --- which is maybe useful for users, but I don't know if it's a pattern that's ever been directly tested/validated with users. I'd recommend doing this just for consistency with other experiences on VA.gov, but if you get any feedback during user research to suggest otherwise let us know!
   - If you go that route, then be sure to adjust the heading structure for the event cards. The H2-styled-as-H3 can just be an H3, etc.
- [x] **Should:** In your prototype and the annotation, "Where: Cheyenne VA Medical Center" is coded as a single heading, rather than just "Where" being a heading. I think the intent here makes sense, but the practical impact may be confusing to screen reader users. The expectation for screen reader users is that a heading describes the section of content that follows it, but here there's no content that comes next. Some screen reader users may interpret it as something being missing or broken even though you're giving users everything you intend to give them. Making the heading "Where" and regular text for the name of the medical center (matching the visual style) is probably the best way to go here.
- [x] **Engineering note:** The prototype doesn't support pagination or sorting, but when the full product is coded there are a couple of things to watch out for:
   - Make sure the "Showing X - Y of Z" message updates to reflect the action the user just took, eg. "Showing 11 - 20 of 1543 events, sorted oldest to newest" or something similar. 
   - When going to another page or sorting the events, make sure focus goes to that "Showing X - Y of Z" message.
   
   What that accomplishes:
   - Keyboard focus moves to the top of the list of currently-displayed events, so the user can immediately start navigating the list they took an action on.
   - Moving focus to the "Showing X - Y" message makes that the first thing screen reader users hear after taking the action, so that message gives them a summary of what's on the page --- and confirmation that the action they took was successfully completed.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [x] Add the Collaboration Cycle initiative epic

**Comment by micahtaylor** [comment]
Moved the numbered results above sort
<img width="1039" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOEpIQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a6635356fa5cc4414233b28b4586bb3e40e1abec/Screenshot%202024-04-09%20at%2012.14.42%E2%80%AFPM.png" alt="Screenshot 2024-04-09 at 12.14.42 PM.png" />

**Comment by micahtaylor** [comment]
Holding off on making the rest of these changes on the prototype while testing. Will bring these items to engineerings attention.

**Comment by micahtaylor** [comment]
Updated headers to reflect feedback. Code can be seen at https://codepen.io/team/VA-gov-Applications/project/editor/DQmqxR
